installer/cli/microk8s.py

```diff
@@ -211,6 +215,37 @@ def kubectl(args) -> int:
         return Linux(args).kubectl()
 
 
+def inspect() -> None:
+    vm_provider_name = "multipass"
+    vm_provider_class = get_provider_for(vm_provider_name)
+    echo = Echo()
+    try:
+        vm_provider_class.ensure_provider()
+        instance = vm_provider_class(echoer=echo)
+        instance.get_instance_info()
+
+        command = ["microk8s.inspect"]
+        output = instance.run(command, hide_output=True)
+        tarball_location = None
+        host_destination = getcwd()
+        if b"Report tarball is at" not in output:
+            echo.error("Report tarball not generated")
+        else:
+            for line in output.split(b"\n"):
+                line = line.decode().strip()
+                if line.startswith("Report tarball is at "):
+                    tarball_location = line.split("Report tarball is at ")[1]
+                    break
+            if not tarball_location:
+                echo.error("Cannot find tarball file location")
+            else:
+                instance.pull_file(name=tarball_location, destination=host_destination)
```

Comment:
We should be printing the inspection output and of course print the path to the tarball (`host_destination`).

Let's echo each line and if we find the `Report tarball is at` line echo the same message with the new location.

Reply:
Just updated the output to look like this:
```
> microk8s inspect
Inspecting Certificates
Inspecting services
  Service snap.microk8s.daemon-cluster-agent is running
  Service snap.microk8s.daemon-containerd is running
  Service snap.microk8s.daemon-apiserver-kicker is running
  Service snap.microk8s.daemon-kubelite is running
  Copy service arguments to the final report tarball
Inspecting AppArmor configuration
Gathering system information
  Copy processes list to the final report tarball
  Copy snap list to the final report tarball
  Copy VM name (or none) to the final report tarball
  Copy disk usage information to the final report tarball
  Copy memory usage information to the final report tarball
  Copy server uptime to the final report tarball
  Copy current linux distribution to the final report tarball
  Copy openSSL information to the final report tarball
  Copy network configuration to the final report tarball
Inspecting kubernetes cluster
  Inspect kubernetes cluster
Inspecting juju
  Inspect Juju
Inspecting kubeflow
  Inspect Kubeflow

Building the report tarball
The report tarball inspection-report-20211118_142318.tar.gz is stored on the current directory
```